packages/manager/modules/pci/src/projects/onboarding/onboarding.routing.js

```diff
@@ -28,23 +30,42 @@ export default /* @ngInject */ ($stateProvider) => {
         $state.go('pci.projects.discovery'),
 
       model: /* @ngInject */ (pciProjectNew, me) =>
-        pciProjectNew.createOrderCart(me.ovhSubsidiary).then((cart) => ({
-          agreements: false,
-          description: 'discoveryProject',
-          cart,
-          hds: cart?.hdsItem !== undefined,
-        })),
+        pciProjectNew.createOrderCart(me.ovhSubsidiary).then((cart) => {
+          const currentDate = new Date();
+          const formatedDate = currentDate.toISOString().split('T')[0];
```

Comment:
**nitpick**: this code does not handle the time zone. For instance, if the client is in France and performs this action at 1AM, the resulting description will be the day before.

Please consider something like this to keep the local time of the client
```javascript
const now = new Date();
[
  now.getFullYear(),
  (now.getMonth() + 1).toString().padStart(2, '0'),
  now.getDate().toString().padStart(2, '0')
].join('-')
```

Or use moment !

Reply:
I used moment.